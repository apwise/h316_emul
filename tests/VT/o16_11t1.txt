ptr ../../tapes/VT/O16-11T1_slst.ptp
g'1
cl
limit 300000000
g'1000
q
