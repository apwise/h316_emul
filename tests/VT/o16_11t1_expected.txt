
MON> ptr ../../tapes/VT/O16-11T1_slst.ptp
MON> g'1

MON> cl
MON> limit 300000000
MON> g'1000

O16-11T1  REV. D  8 MAY 1970
END OF PASS 000005000
END OF PASS 000010000
0305643329: limit reached

MON> q
